deploy/template.yaml

```diff
@@ -221,10 +233,16 @@ objects:
             httpGet:
               path: /live
               port: 8080
+            failureThreshold: ${LIVENESS_PROBE_FAILURE_THRESHOLD}
```

Comment:
sorry, my bad but I think this should be with double braces `${{LIVENESS_PROBE_FAILURE_THRESHOLD}}` (all of these parameters are int)